README.md

```diff
@@ -58,3 +58,188 @@ select * where {
     <urn:a> ?p ?o  .
 }
 ```
+
+# Secured Kafka Sink with FailedRecordProducer Configuration
+
+This guide provides instructions on how to start a secured Kafka Sink while configuring
+
+the inner FailedRecordProducer with security properties. The configuration can be provided
+
+either using properties with a "producer.override." prefix or through environment variables in the Dockerfile.
+
+## Configuration Options
+
+### Using "producer.override." Prefix in Properties
+
+To configure the inner FailedRecordProducer with security properties using a "producer.override." prefix:
+
+Provide the `graphdb-kafka-sink.properties` file to include security properties with the "producer.override." prefix. For example:
+
+```properties
+
+name=GraphDBSinkConnector
+connector.class=com.ontotext.kafka.GraphDBSinkConnector
+
+graphdb.server.url=Type: String;\nDescription: GraphDB Server URL
+graphdb.server.repository=Type: String;\nDescription: GraphDB Repository
+
+graphdb.batch.size=Type: Int;
+
+graphdb.auth.type=Type: enum[NONE, BASIC, CUSTOM];\nDescription: Authentication type (default NONE)
+graphdb.auth.basic.username=Type: String;
```

Comment:
Do we need to include all those? Isn't it better to just mention the properties for SASL/SSL that need to be added?

Reply:
This is an example of a configuration file. This properties are really important for our kafka-sink connector to work